core/src/main/java/org/testcontainers/utility/AuditLogger.java

```diff
@@ -25,14 +28,28 @@
     private static final ObjectMapper objectMapper = new ObjectMapper();
     public static final String MDC_PREFIX = AuditLogger.class.getCanonicalName();
 
-    public static void doLog(String action, String image, String containerId, DockerCmd<?> cmd) {
+    public static void doLog(@NotNull String action,
+                             @Nullable String image,
+                             @Nullable String containerId,
+                             @NotNull DockerCmd<?> cmd) {
+
         doLog(action, image, containerId, cmd, null);
     }
 
-    public static void doLog(String action, String image, String containerId, DockerCmd<?> cmd, @Nullable Exception e) {
-        MDC.put(MDC_PREFIX + ".Action", action);
-        MDC.put(MDC_PREFIX + ".Image", image);
-        MDC.put(MDC_PREFIX + ".ContainerId", containerId);
+    public static void doLog(@NotNull String action,
+                             @Nullable String image,
+                             @Nullable String containerId,
+                             @NotNull DockerCmd<?> cmd,
+                             @Nullable Exception e) {
+
+        if (! log.isTraceEnabled()) {
```

Comment:
Fine, OK 👍 
Let's merge #362 first to avoid spending too much time resolving conflicts on the `TestcontainersConfiguration` class